packages/documentation/copy/en/reference/Advanced Types.md

```diff
@@ -3,7 +3,7 @@ title: Advanced Types
 layout: docs
 permalink: /docs/handbook/advanced-types.html
 oneline: Advanced concepts around types in TypeScript
-deprecated_by: /docs/handbook/2/types-from-types.html
```

Comment:
The old page covers a ton of different kinds of types, but here you're linking to one very specific concept on a page that's mostly about narrowing and doesn't really serve the same purpose.